public/app/features/alerting/unified/Analytics.ts

```diff
@@ -26,38 +26,38 @@ export const LogMessages = {
   unknownMessageFromError: 'unknown messageFromError',
 };
 
-const alertingLogger = createMonitoringLogger('features.alerting', { module: 'Alerting' });
+const { logInfo, logError, logMeasurement } = createMonitoringLogger('features.alerting', { module: 'Alerting' });
 
-export function logInfo(message: string, context?: Record<string, string>) {
-  alertingLogger.logInfo(message, context);
-}
-
-export function logError(error: Error, context?: Record<string, string>) {
-  alertingLogger.logError(error, context);
-}
+export { logInfo, logError, logMeasurement };
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function withPerformanceLogging<TFunc extends (...args: any[]) => Promise<any>>(
+  type: string,
   func: TFunc,
-  message: string,
   context: Record<string, string>
 ): (...args: Parameters<TFunc>) => Promise<Awaited<ReturnType<TFunc>>> {
   return async function (...args) {
     const startLoadingTs = performance.now();
+
     const response = await func(...args);
-    logInfo(message, {
-      loadTimeMs: (performance.now() - startLoadingTs).toFixed(0),
-      ...context,
-    });
+    const loadTimesMs = Number((performance.now() - startLoadingTs).toFixed(0));
```

Comment:
I believe the double conversion from number to string and from string to number is not needed.
It was necessary to convert number to string because the `logInfo` function doesn't accept anything other than string values

Reply:
Yep, this is definitely unnecessary 🤦